velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -63,6 +68,9 @@ class SparkCastHooks : public exec::CastHooks {
   exec::PolicyType getPolicy() const override;
 
  private:
+  template <typename T>
+  Expected<Timestamp> castNumberToTimestamp(T value) const;
```

Comment:
Perhaps document this function, including the supported types.

Reply:
done